server/eb/Makefile

```diff
@@ -30,6 +30,9 @@ build: clean
 	if [ -d scripts ] ; then \
 	  cp -r scripts/* artifact.dir/server/common/web/templates;  \
 	fi; \
+	if [ -d plugins ] ; then \
```

Comment:
There is now a customization directory that all these things are placed in.
You will need to make sure that this gets updated to  [ -d customize/plugins ]